tests/headless/test_cdn.py

```diff
@@ -241,8 +241,13 @@ def test_not_cached_403(base_url, is_behind_cdn, slug):
      '/en-US/docs/top-level',
      '/en-US/docs/with-errors',
      '/en-US/docs/without-parent'])
-def test_cached(base_url, is_behind_cdn, slug):
+def test_cached(base_url, is_behind_cdn, is_local_url, is_searchable, slug):
     """Ensure that these requests that should return 200 are cached."""
+    if is_local_url:
+        if any(slug.startswith(p) for p in ('/diagrams/', '/presentations/')):
+            pytest.xfail('not yet available in the sample database')
```

Comment:
These are the [legacy files](https://github.com/mozilla/kuma/blob/df5bc3b1bcc2addc54915b8362e99c86fa50166f/kuma/urls.py#L128-L135), not part of the sample database. We could add ``SERVE_LEGACY`` to the information in ``_kuma_status.json``, or just assume they aren't available. But they won't be added to the sample DB.